docs/source/guides/staking_guide.rst

```diff
@@ -1,8 +1,19 @@
+.. _staking-guide:
+
 =======================
 NuCypher Staking Guide
 =======================
 
-All staking-related operations are performed through the ``nucypher stake`` command:
+The account which is actively doing staking operations is necessary to be a hot
```

Comment:
What are we defining as "staking operations" here? Staking operations as I understand them (e.g. stake, split stake, etc.) do not require a hot wallet. If you're referring to Worker operations (e.g. re-encrypt) I'd call them something else.

Reply:
True, that's only stakeholder's commands. I thought to put worker commands here, but I realized that there is already a section for that. Will fix